src/client/pages/Offer/Introduction/DetailsModal/Details/components/Details.styles.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from '@emotion/styled'
```

Comment:
Exact copy of `OfferNew/Introduction/DetailsModal/Details/components/Details.styles.tsx `